src/pages/HomePage/HomePage.js

```diff
@@ -1,71 +1,58 @@
 import React from 'react'
 import SubHeader from '../../components/SubHeader/SubHeader'
 import Carousel from '../../components/Carousel/Carousel'
-import categoryImage1 from '../../assets/Images/skincare-2.jpg'
-import categoryImage2 from '../../assets/Images/tampons.jpg'
-import categoryImage3 from '../../assets/Images/makeup-img-2.jpg'
-import categoryImage4 from '../../assets/Images/hair-care-img.jpg'
-import categoryImage5 from '../../assets/Images/body-care-img.jpg'
+
 import brandImage1 from '../../assets/Images/loreal_logo.jpg'
 import brandImage2 from '../../assets/Images/lakme_logo.jpg'
 import brandImage4 from '../../assets/Images/maybelline_logo.png'
-import brandImage5 from '../../assets/Images/Wishper_logo.png'
+import brandImage5 from '../../assets/Images/mamaearth-logo.jpg'
+import brandImage6 from '../../assets/Images/wow-logo.png'
+import brandImage7 from '../../assets/Images/Wishper_logo.png'
 import offerImage1 from '../../assets/Images/offerImage1.jpg'
 import offerImage2 from '../../assets/Images/offerImage2.jpg'
 import { Link } from 'react-router-dom'
 import './HomePage.css'
 import Footer from '../../components/Footer/Footer'
+import { useContext } from 'react'
+import { StateContext } from '../../Context'
+import { useEffect } from 'react'
```

Comment:
You can write the 2 modules of useContext and useEffect together too `import {useContext,useEffect} from "react"`